From 4f4b18b2ec8de2e5efed9625f1a9fbe82e8f2e73 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Mon, 4 Mar 2013 07:38:08 +0100
Subject: BUILD/MINOR: syscall: add definition of NR_accept4 for ARM

This platform was not covered and older libc do not provide accept4().
---
 include/common/syscall.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/common/syscall.h b/include/common/syscall.h
index 659b0ca..0b1ccd6 100644
--- a/include/common/syscall.h
+++ b/include/common/syscall.h
@@ -132,6 +132,8 @@
 #define __NR_accept4            288
 #elif defined(__sparc__) || defined(__sparc64__)
 #define __NR_accept4            323
+#elif defined(__arm__) || defined(__thumb__)
+#define __NR_accept4            (__NR_SYSCALL_BASE+366)
 #else
 #define ACCEPT4_USE_SOCKETCALL    1
 #ifndef SYS_ACCEPT4
-- 
1.7.1

